src/components/Blogs/BlogItem.module.scss

```diff
@@ -0,0 +1,76 @@
+.blog_item {
+    background-color: #fff;
+    color: #37474F;
```

Comment:
This color should be coming from **globals.css** variables.

Reply:
solved